curriculum/challenges/english/25-front-end-development/lab-colored-boxes/66f3f6eb66ea9dc41cdc30df.md

```diff
@@ -0,0 +1,210 @@
+---
+id: 66f3f6eb66ea9dc41cdc30df
+title: Design a Set of Colored Boxes
+challengeType: 14
+dashedName: lab-colored-boxes
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you'll practice using CSS colors by designing boxes.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab.
+
+**User Stories:**
+
+1. You should set the background color for `body` to `#f4f4f4`.
+2. You should have a `div` with a class of `color-grid` to hold all your color elements.
+3. You should have five `div` elements within `color-grid` `div`.
```

Comment:
```suggestion
3. You should have five `div` elements within `div.color-grid`.
```

Reply:
I personally would use ``You should have five `div` elements within the `.color-grid` element.``